atomic_reactor/plugins/post_koji_import.py

```diff
@@ -157,7 +161,7 @@ def _iter_work_metadata_outputs(
         :type _filter: dict[str, any] or None
         :return: an iterator that yields a tuple in form (platform, output).
         """
-        for worker_platform, metadata in self._worker_metadatas.items():
+        for worker_platform, metadata in self._builds_metadatas.items():
```

Comment:
how about `image_platform` or `build_platform` for clarity?

Reply:
:sweat_smile: You are right.